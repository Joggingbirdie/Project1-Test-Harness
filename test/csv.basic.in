Name, Age, Score
Alice, 28, 95
Bob, 25, 89
Charlie, 32, 75
