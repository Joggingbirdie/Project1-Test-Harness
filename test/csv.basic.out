Sum of Age: 85
Sum of Score: 259
